src/main/java/com/toptrumps/core/card/Card.java

```diff
@@ -0,0 +1,46 @@
+package com.toptrumps.core.card;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.function.Supplier;
+
+public class Card {
+
+    final private String name;
+    private ArrayList<Attribute> attributes;
+
+    /**
+     * Lazily filter the highest attribute and store it as a final member variable
+     */
+    final private Supplier<Attribute> highestAttribute = () -> Collections.max(attributes);
+
+    public Card(String name, ArrayList<Attribute> attributes) {
+        this.name = name;
+        this.attributes = attributes;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    // TODO: Revisit this method to increase efficiency
```

Comment:
Possibly consider HashMaps! Time complexity of O(1)

Reply:
I sure will! :)